daemon/algod/api/algod.oas3.yml

```diff
@@ -2817,6 +2837,93 @@
         "tags": [
           "private"
         ]
+      },
+      "post": {
+        "description": "Given a participation id, append state proof keys to a particular set of participation keys.",
```

Comment:
1. `id` -> `ID`
2. drop the "." at the end of the sentence ( just to align with the rest of the descriptions )

Reply:
Looks like the descriptions are almost equally divided! 158 with periods, 192 without. I'll save that exercise for another PR, there are probably other things that could make the verbiage more consistent in addition to punctuation.